toolkit/tools/pkg/imagecustomizerlib/customizeutils.go

```diff
@@ -134,6 +140,26 @@ func deleteResolvConf(imageChroot *safechroot.Chroot) error {
 	return err
 }
 
+func refreshInitrd(partitionsCustomized bool, imageChroot *safechroot.Chroot) error {
```

Comment:
While not common, there are images that don't contain the initramfs package. Might want to check if mkinitrd is pressent at all.